drivers/sshj/src/main/java/org/jclouds/sshj/SshjSshClient.java

```diff
@@ -460,7 +460,7 @@ public ExecResponse create() throws Exception {
             Command output = session.exec(checkNotNull(command, "command"));
             String outputString = IOUtils.readFully(output.getInputStream()).toString();
             output.join(sshClientConnection.getSessionTimeout(), TimeUnit.MILLISECONDS);
-            int errorStatus = output.getExitStatus();
+            int errorStatus = output.getExitStatus() != null ? output.getExitStatus() : -1;
```

Comment:
Move the logic to the [ExecResponse constructor](https://github.com/devcsrj/jclouds/blob/master/compute/src/main/java/org/jclouds/compute/domain/ExecResponse.java#L29-L33)? This way the approach would be ssh driver agnostic.